packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -1156,9 +1156,7 @@ class MemoryTestRunner extends TestRunner {
   protected async initPersistence(
     serializer: JsonProtoSerializer
   ): Promise<Persistence> {
-    const persistence = new MemoryPersistence(this.clientId);
-    await persistence.start();
-    return persistence;
+    return new MemoryPersistence(this.clientId);
```

Comment:
This is kind of weird. Care to make the function not async and use a `Promise.resolve()` here?

Reply:
Sure.